### PR TITLE
Add player health event, german translation and version key

### PR DESCRIPTION
**Pull request by davidramiro**
Hi, 

first of all, thanks for the great work! Added and tested a few more events. It's my first contribution on a HA component, so not sure if everything is fine in terms of conventions/style. If all is well, I would love to contribute even more, for example implement #5.

### CODE REVIEW

File: info.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Open that directory and add a file called **gamestate_integration_homeassistant.
 Add the following content and replace the "uri" with the one copied during the integration setup:
 
 ```
-"HomeAssistant Integration v.1"
+"HomeAssistant Integration v.2"
```
Comment (lociii) [inline info.md:24]: Please change to v0.2 as you defined in the manifest

**Review by lociii — CHANGES_REQUESTED** [review]

**Comment by lociii** [comment]
Great work. Thanks.
As I am not playing CS:GO actively anymore, I just reviewed the code and didn't test it.

File: info.md (hunk 1 above)
Comment (davidramiro) [reply info.md]: Good catch. I only later discovered your release versioning, so manifest, changelog and and this cfg now all follow the SemVer scheme.

**Comment by davidramiro** [comment]
Should be all set, cfg resembles the MINOR update to 1.2.0 now.

**Comment by lociii** [comment]
Thanks for your contribution!

**Pull request merged**